public/app/core/logs_model.ts

```diff
@@ -445,6 +459,15 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefi
   };
 }
 
+function hasFields(logSeries: DataFrame[]): boolean {
+  for (const series of logSeries) {
```

Comment:
i think this function could be simplified by just returning
`logSeries.some(series => series.fields.length)`